2_deploy_ses_aio/roles/setup_ses_configs/tasks/main.yml

```diff
@@ -0,0 +1,24 @@
+---
+- name: Ensure ses config directory exist
+  file:
+    path: "{{ ses_config_path }}"
+    state: directory
+  when:
+    - ses_openstack_config
+
+- name: Drop ceph conf and keys
+  copy:
+    dest: "{{ ses_config_path }}/{{ item.item }}"
+    content: "{{ item.content | b64decode }}"
+    mode: 0644
+  with_items:
```

Comment:
Where does .conf_options or .ceph_files come from?

Reply:
It's set by 2_deploy_ses_aio/roles/ses/tasks/openstack_config.yml